airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/query.py

```diff
@@ -1984,3 +1984,174 @@ def record_process_components(self, record: MutableMapping[str, Any]) -> Iterabl
         record.pop("inventoryItem", None)
 
         yield record
+
+
+class OrderRisk(ShopifyBulkQuery):
+    """
```

Comment:
why this doc string?

Reply:
It contains the final `query` look and structure, any issues having it's there?